src/reconstruction/refinement/removeUnusedGenes.m

```diff
@@ -11,21 +11,18 @@
 %
 % OUTPUT:
 %		 model:    COBRA model structure with updated gene field reflecting only
-%							 content present in the model
+%                  content present in the model
```

Comment:
yes modelNew is the output. There were some tabs left in the file. removed them and updated the docu